handbook/design/design-process.md

```diff
@@ -0,0 +1,121 @@
+    *   Competitive designs
+    *   Findings publishing and review
+        *   Findings should be recorded in google docs and announced in the corresponding team channel or #product on Slack
+            *   Research documents should contain a clear summary of findings and charts of supporting data are helpful.
+        *   If the size of the project requires, a review meeting should be held to discuss key findings.
+    *   Process
+        *   A GitHub issue should be created to track the work
+        *   Research is conducted according to requirements
+        *   Publish research to google drive
+        *   Announce research in slack
+        *   Schedule and conduct a presentation meeting with stakeholders
+        *   Stakeholders will approve the GitHub issue indicating the designs are ready for the next phase
+    *   Tools
+        *   Google Docs
+        *   Google forms for polls
+        *   Usertesting.com
+*   Wireframes
+    *   Wireframes should utilize the Sourcegraph Figma component system
+    *   Wireframes should generally utilize realistic data and text for each element. In cases where many elements are required, ‘lorem epsum’ placeholder text will suffice if the initial or prominent elements are real data.
+    *   Process
+        *   A GitHub issue should be created to track the work
+            *   Label the issue with the design and/or ux label
+            *   TBD - how do we replicate pull request reviewers? 
+        *   Wireframes are produced in Figma
+        *   Wireframes are announced in Slack and linked in the GitHub issue well before they are complete for review
+        *   WIreframes are hallway tested with internal users and product managers
+        *   Schedule and conduct a presentation meeting
+        *   Stakeholders will approve the GitHub issue indicating the designs are ready for the next phase
+    *   Checklist
+        *   Has the change’s effect on the CLI been considered?
+            *   Does the language in the UI map to the language in the CLI?
+        *   Enterprise / Cloud
+        *   Signed in vs. anonymous user
+        *   User permissions
+        *   Interactive mode / plain text mode
+        *   TBD - add a standard UX checklist
+    *   Tools
+        *   Figma
+*   Testing
+    *   Process
+        *   A GitHub issue should be created to track the work
+        *   Testing is conducted according to requirements
+        *   Publish testing results summary to google drive
+        *   Announce results in slack
+        *   Create a GitHub issue to address items in the test results
+    *   Tools
+        *   Maze.design
+        *   UserTesting.org
+*   Visual design
+    *   Visual design should utilize the Sourcegraph’s Figma based component system
+    *   Dark compositions will be created for each major design
+    *   If new components are required, components will be created in the file itself, not the component system
+    *   Design checklist
```

Comment:
Didn't we add a checklist somewhere else? Should we move it here or link to it?

Reply:
Thanks, I moved it.